CHANGELOG.md

```diff
@@ -1,5 +1,93 @@
+
+### Components
+Updated Components
+* ruby (2.2.5 -> 2.2.6)
+* rubygems (2.6.6 -> 2.6.8)
+* omnibus-ctl (a0ccf08a -> 4db34135)
+* server-jre (8u91 -> 8u121)
+* knife-opc (a08237d4 -> 00a6866c)
+* berkshelf-no-depselector (abb27143 -> f49321c9)
+* ohai (c34212ea -> e5abf16c)
+* appbundler (76cb1728 -> 6582b688)
+* rb-readline (5e1c55db -> 323fed5a)
+* chef_backup-gem (67b1f51a -> bc1105b7)
+* chef (c21db758 -> fcb0ccc2)
+
+### Contributors
+* Steven Danna
+* Stephan Renatus
+* Mark Harrison
+* Thom May
+* Marc Paradise
+* Larry Eichenbaum
+* Matt Campbell
+* Shadae Holmes
+* Bryan McLellan
+* Nathan L Smith
+* Ryan Cragun
+* Sean Nolen
+
+**Fixed bugs:**
+
+- chef-server-ctl password command does not accept special characters  [\#366](https://github.com/chef/chef-server/issues/366)
+- `chef-server-ctl cleanse` removes plugin config, breaking reconfigure [\#115](https://github.com/chef/chef-server/issues/115)
+- Chef Server API should not allow usernames with spaces [\#90](https://github.com/chef/chef-server/issues/90)
+- Confusing error message with duplicate email address [\#59](https://github.com/chef/chef-server/issues/59)
+
+**Closed issues:**
+
+- enable fips mode with  chef-server-fips-core package [\#1024](https://github.com/chef/chef-server/issues/1024)
+- error in chef-server-ctl reconfigure in docker. [\#960](https://github.com/chef/chef-server/issues/960)
+- Data bag search doesn't work correctly for encrypted arrays [\#876](https://github.com/chef/chef-server/issues/876)
+- rebar3 warning during build [\#630](https://github.com/chef/chef-server/issues/630)
+- oc\_id: email configuration [\#547](https://github.com/chef/chef-server/issues/547)
+- Add settings to production.yml for sending mail [\#185](https://github.com/chef/chef-server/issues/185)
+
+**Merged pull requests:**
+
+- Bump omnibus-software to roll back to runit 2.1.1 [\#1063](https://github.com/chef/chef-server/pull/1063) ([stevendanna](https://github.com/stevendanna))
+- Minor README changes [\#1062](https://github.com/chef/chef-server/pull/1062) ([stevendanna](https://github.com/stevendanna))
+- Update omnibus-software to get the latest server-jre [\#1059](https://github.com/chef/chef-server/pull/1059) ([stevendanna](https://github.com/stevendanna))
+- Fix the fips build [\#1056](https://github.com/chef/chef-server/pull/1056) ([stevendanna](https://github.com/stevendanna))
+- Remove coverdata files from oc\_erchef dir [\#1055](https://github.com/chef/chef-server/pull/1055) ([stevendanna](https://github.com/stevendanna))
+- Fix Makefiles to not distclean by default [\#1054](https://github.com/chef/chef-server/pull/1054) ([stevendanna](https://github.com/stevendanna))
+- Update depselector\_rb Gemfile to use HTTPS [\#1053](https://github.com/chef/chef-server/pull/1053) ([stevendanna](https://github.com/stevendanna))
+- Revert ruby to 2.2.6 [\#1052](https://github.com/chef/chef-server/pull/1052) ([stevendanna](https://github.com/stevendanna))
```

Comment:
May be worth editing out this one and the upgrade we reverted from, to keep it easier to scan